### PR TITLE
Debian formatted copyright file for packages

### PR DESCRIPTION
**Pull request by asmorkalov**

### CODE REVIEW

**Comment by asmorkalov** [comment]
@ilya-lavrenov please review.

File: cmake/OpenCVPackaging.cmake [hunk 1]
```diff
@@ -214,6 +215,12 @@ if(CPACK_GENERATOR STREQUAL "DEB")
             DESTINATION "share/doc/${CPACK_DEBIAN_COMPONENT_${comp_upcase}_NAME}"
             COMPONENT "${comp}")
 
+    if(OPENCV_DEBIAN_COPYRIGHT_FILE)
+        install(FILES "${TEGRA_DIR}/debian/copyright"
```
Comment (jet47) [inline cmake/OpenCVPackaging.cmake]: It should be `${OPENCV_DEBIAN_COPYRIGHT_FILE}`.

**Comment by ilya-lavrenov** [comment]
@asmorkalov, please, fix trailing spaces issue http://pullrequest.opencv.org/buildbot/builders/precommit_docs/builds/4408/steps/whitespace/logs/stdio

**Comment by asmorkalov** [comment]
Done.

**Comment by ilya-lavrenov** [comment]
:+1:

**Comment by alalek** [comment]
@asmorkalov @vpisarev Please take a look on the copyright file.
Looks like it is specified during build machine only. Do we need it in the repo? (we already have something here https://github.com/Itseez/opencv/blob/2.4/LICENSE)

**Comment by asmorkalov** [comment]
The new file is formatted according to Debian package roles. Of cause I can implement automated formatter for, but it's easier to do it once manually and do not worry at all.

**Pull request merged**